/home/pmos/build/src/android_kernel_huawei_alice-1ecb135f6/net/core/sock.c: In function 'sock_setbindtodevice':
/home/pmos/build/src/android_kernel_huawei_alice-1ecb135f6/net/core/sock.c:578:61: error: 'AID_INET' undeclared (first use in this function); did you mean 'AF_INET'?
  578 |  if (!ns_capable(net->user_ns, CAP_NET_RAW) && !in_egroup_p(AID_INET))
      |                                                             ^~~~~~~~
      |                                                             AF_INET



diff --git a/net/core/sock.c b/net/core/sock.c
index c7f705a0..35019d80 100644
--- a/net/core/sock.c
+++ b/net/core/sock.c
@@ -575,7 +575,7 @@ static int sock_setbindtodevice(struct sock *sk, char __user *optval,
 	/* < DTS2015012106130 guoxiaojie 00276951 2015.01.22
 		double cell concurrent download feature */
 	ret = -EPERM;
-	if (!ns_capable(net->user_ns, CAP_NET_RAW) && !in_egroup_p(AID_INET))
+	if (!ns_capable(net->user_ns, CAP_NET_RAW))
 		goto out;
 	/* DTS2015012106130 guoxiaojie 00276951 2015.01.22
 		double cell concurrent download feature > */
